lang/smaily-et.po

```diff
@@ -372,14 +441,24 @@ msgstr "Smaily kasutajatunnused salvestatud!"
 msgid "Something went wrong with saving data!"
 msgstr "Midagi läks valesti andmete salvestamisel!"
 
-#: inc/Base/Upgrade.php:101
+#: inc/Base/Upgrade.php:104
 msgid ""
 "Smaily for Woocommerce plugin has changed abandoned cart exported fields.\n"
 "\t\t\t\tPlease check your plugin settings!"
 msgstr ""
 "Smaily WooCommerce-le on muutnud unustatud ostukorvi lisaväljasid. Palun "
 "kontrollige pistikmooduli sätteid!"
 
+#: inc/Base/Upgrade.php:114
+msgid ""
+"Smaily for Woocommerce plugin has changed checkout subscription checkbox "
+"behaviour.\n"
+"\t\t\t\tPlease check your plugin settings!"
+msgstr ""
+"Smaily WooCommerce-le on muutnud tellimuslehel uudiskirjaga liitumise "
```

Comment:
I wouldn't translate the name of the plugin. It is totally OK, if it remains "Smaily for WooCommerce" in Estonian.

Reply:
Changed name translation everywhere to `Smaily for WooCommerce` https://github.com/sendsmaily/smaily-woocommerce-plugin/pull/45/commits/2e24bba061d5411cf28d739eab9e26a7f36d7ca7#